match/lib/match/runner.rb

```diff
@@ -46,13 +46,20 @@ def run(params)
         google_cloud_bucket_name: params[:google_cloud_bucket_name].to_s,
         google_cloud_keys_file: params[:google_cloud_keys_file].to_s
       })
+
+      UI.message("====================================")
+      UI.message("       Download from Storage        ")
+      UI.message("====================================")
       storage.download
 
       # Init the encryption only after the `storage.download` was called to have the right working directory
       encryption = Encryption.for_storage_mode(params[:storage_mode], {
         git_url: params[:git_url],
         working_directory: storage.working_directory
       })
+      UI.message("====================================") if encryption
```

Comment:
Could you please write this as:

```ruby
if encryption
  UI.message("====================================")
  UI.message("           Decrypt files            ")
  UI.message("====================================")
  encryption.decrypt_files
end
```
___

Also, do you think we could use the [`UI.header`](https://github.com/fastlane/fastlane/blob/f32b007ff45e648b37b6c9c2037ac481f36b7780/fastlane_core/lib/fastlane_core/ui/implementations/shell.rb#L86) function instead?

Reply:
Oh yes, very much. I just did this while creating #13943 but didn't want to just throw it away, so parked it here in this branch (and PR so I don't forget about it).

Reply:
Ahh, gotcha - sounds good to me! :)